system/keyboardlayouts/croatian.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+Please use English language names instead.
+Default font lacks support for all characters
+-->
+<keyboardlayouts>
+  <layout language="Croatian" layout="QWERTY">
+    <keyboard>
+      <row>1234567890'+</row>
+      <row>qwertzuiopšđ</row>
+      <row>asdfghjklčćž:</row>
+      <row>yxcvbnm,.-</row>
+    </keyboard>
+    <keyboard modifiers="shift">
+      <row>!"#$%&/()=?*</row>
```

Comment:
Ok i have made update, sorry about that i did never made any modification to kodi so this is.my first request